src/overlays/actors/ovl_En_In/z_en_in.h

```diff
@@ -28,7 +28,7 @@ typedef struct EnIn {
     /* 0x238 */ char unk238[0x4];
     /* 0x23C */ u8 unk23C;
     /* 0x23D */ u8 unk23D;
-    /* 0x240 */ u8 *unk240;
+    /* 0x240 */ Path *path;
```

Comment:
Our style is `Type* varname`; the format script doesn't do headers which is why it didn't catch this, presumably.